DirectX11/FrameAnalysis.cpp

```diff
@@ -1997,18 +2000,18 @@ HRESULT FrameAnalysisContext::FrameAnalysisFilename(wchar_t *filename, size_t si
 		StringCchPrintfExW(pos, rem, &pos, &rem, NULL, L"@%p", handle);
 
 	if (compute) {
-		StringCchPrintfExW(pos, rem, &pos, &rem, NULL, L"-cs=%016I64x", mCurrentComputeShader);
+		StringCchPrintfExW(pos, rem, &pos, &rem, NULL, L"-cs=%016I64x", G->mComputeShaders.mCurrent);
```

Comment:
Why is this moved to Globals? Assigned shaders are per-context and definitely not global.